src/fiat-connect-client.ts

```diff
@@ -163,6 +178,26 @@ export class FiatConnectClient implements FiatConnectApiClient {
     }
   }
 
+  /**
+   * Returns an approximation of the current server time, taking into account clock differences
+   * between client and server. Returns the earliest possible server time based on the max error
+   * of the clock diff between client and server, to ensure that sessions created using this time
+   * are not issued in the future with respect to the server clock.
+   */
+  async getServerTime(): Promise<Result<Date, ResponseError>> {
```

Comment:
maybe `getServerTimeApprox`, similar to `getClockDiffApprox`?

Reply:
Yeah, I like this better